deploy/lib/table-manager.libsonnet

```diff
@@ -0,0 +1,73 @@
+local kube = import 'kube-libsonnet/kube.libsonnet';
+
+{
+
+    table_manager_deployment:
+        local name = $._config.tableManager.name;
+        local labels = $._config.tableManager.labels;
+
+        # SchemaConfig volume
+        local schemaConfigVolume = kube.ConfigMapVolume($.schema_configmap);
+        local schemaConfigVolumeMount = {
+            config_volume: {
+                mountPath: '/etc/cortex',
+                readOnly: true
+            },
+        };
+
+        # Ports
+        local tableManagerPorts = {
+            http: {
+                containerPort: 80
+            },
+        };
+
+        # Arguments
+        local extraArgs = $._config.tableManager.extraArgs;
+        local args = [
+            '-config-yaml=/etc/cortex/schemaConfig.yaml',
+        ];
+
+        # Environment Variables
+        local env = $._config.tableManager.env;
+        local envKVMixin = $._config.tableManager.envKVMixin;
+        local extraEnv = [{name: key, value: extraEnv[key]} for key in std.objectFields(envKVMixin)];
```

Comment:
should be: `local extraEnv = [{name: key, value: envKVMixin[key]} for key in std.objectFields(envKVMixin)];`